qiita_ware/util.py

```diff
@@ -117,104 +116,14 @@ def stats_from_df(df):
     out = {}
 
```

Comment:
the return description for this method says the keys are both names of metadata categories, and tuples, and the code suggests the keys are strings. possible to clarify this?